genericjpa/common-apis/src/main/java/org/hibernate/search/genericjpa/annotations/InIndex.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.genericjpa.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * This annotation is needed to specify that a class is located in an index. <br>
+ * <br>
+ * This is only needed (but always allowed), if objects of subclasses are supplied to indexing related operations. Then
+ * the "least top level" class that is annotated with @InIndex is used. operations
```

Comment:
This is just in here because it is needed in the SubclassSupportInitializer which is in turn used in StandaloneSearchConfiguration. This is a "relic" from genericjpa which I don't really want to remove now because I would have change code now and revert these changes for my genericjpa code. I hope this is okay. (btw all classes which are needed exclusively for genericjpa should be in genericjpa packages).